models/repo/search.go

```diff
@@ -5,20 +5,70 @@ package repo
 
 import "code.gitea.io/gitea/models/db"
 
+// Strings for sorting result
+const (
+	SearchOrderByAlphabetically        db.SearchOrderBy = "owner_name ASC, name ASC"
```

Comment:
this will affect the explore page similar as  #30784 too

Reply:
if desired I'll create a dedicated backport for 1.22 cc @lunny

Reply:
Sorry, I don't think we should backport this to v1.22. Please only backport bugfix since it's a refactor PR.

Reply:
@lunny I'll create the pull so you do understand what I ment ...

Reply:
-> #30882